include/openenclave/enclave.h

```diff
@@ -386,6 +386,13 @@ oe_result_t oe_verify_report(
     size_t report_size,
     oe_report_t* parsed_report);
 
+struct oe_eeid_t_;
+oe_result_t oe_verify_report_eeid(
```

Comment:
Since this API name isn't aligned with the future direction of terminology (it uses "report" vs "evidence" or "attestation result"), it should be marked as @experimental so that we can make breaking changes at will.